DOCUMENTATION.md

```diff
@@ -384,17 +422,57 @@ try {
 } catch (NotifyException $e){}
 ```
 
-</details>
+### Arguments
+
+#### templateId (required)
+
+Sign in to GOV.UK Notify and go to the __Templates page__ to find the template ID.
+
+#### personalisation (required)
+
+The personalisation argument always contains the following required parameters for the letter recipient's address:
+
+- `address_line_1`
+- `address_line_2`
+- `postcode`
+
+Any other placeholder fields included in the letter template also count as required parameters. You need to provide their values in a dictionary with key value pairs. For example:
+
+```php
+personalisation=
```

Comment:
variables should start with $, and all statements (ie - the last line of a code snippet) should end with a semicolon

```php
$personalisation = [
....
];
```